### PR TITLE
Add topic getter to message class

### PR DESCRIPTION
**Pull request by rwstauner**

### CODE REVIEW

File: spec/pulsar/client_spec.rb [hunk 1]
```diff
@@ -43,9 +43,12 @@
 
     it "can consume a single topic" do
```
Comment (atezs82) [inline spec/pulsar/client_spec.rb:44]: We shall have a test case for the nil return value as well.

File: ext/bindings/message.hpp [hunk 2]
```diff
@@ -21,6 +21,7 @@ namespace pulsar_rb {
   class Message {
   public:
     pulsar::Message _msg;
+    bool built = false; // built with builder rather than returned from c++
```
Comment (atezs82) [inline ext/bindings/message.hpp:24]: Given the comment you have written for getTopicName above, shall we name this `received` instead? What do you think?

File: spec/pulsar/client_spec.rb (hunk 1 above)
Comment (rwstauner) [reply spec/pulsar/client_spec.rb:44]: that's in the message spec.

File: ext/bindings/message.hpp (hunk 2 above)
Comment (rwstauner) [reply ext/bindings/message.hpp]: name it `received`, default it  to `true`, and set it to `false` when we do the building?
sure.

Comment (atezs82) [reply ext/bindings/message.hpp]: Yes, that should do it.

**Review by atezs82 — APPROVED** [review]

**Pull request merged**